.github/workflows/integration.yaml

```
@@ -0,0 +1,43 @@
+name: Tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * TUE'
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
```

I'm returning the favour :)  Shall we un-name the common one-liner?
```suggestion
    - uses: actions/checkout@v2
```